tests/sentry/integrations/vsts/test_issues.py

```diff
@@ -199,29 +197,6 @@ def test_create_issue(self, mock_record):
             {"op": "add", "path": "/fields/System.Description", "value": "<p>Fix this.</p>\n"},
             {"op": "add", "path": "/fields/System.History", "value": "<p>Fix this.</p>\n"},
         ]
-        assert len(mock_record.mock_calls) == 2
-        start, halt = mock_record.mock_calls
-        assert start.args[0] == EventLifecycleOutcome.STARTED
-        assert halt.args[0] == EventLifecycleOutcome.SUCCESS
-
-    @responses.activate
-    @patch("sentry.integrations.utils.metrics.EventLifecycle.record_event")
-    def test_create_issue_failure(self, mock_record):
```

Comment:
Even though these are no longer used for metrics, do you think its valuable to keep these tests around? Or do we have coverage elsewhere?